src/theme/shared/shadows.ts

```diff
@@ -4,6 +4,7 @@ const shadows = {
   sm: '0px 4px 12px rgba(0, 0, 0, 0.05), 0px 8px 24px rgba(0, 0, 0, 0.1);',
   md: '0px 2px 6px rgba(0, 0, 0, 0.05), 0px 8px 24px rgba(0, 0, 0, 0.1), 0px 20px 48px rgba(0, 0, 0, 0.15)',
   lg: '0px 2px 8px rgba(0, 0, 0, 0.04), 0px 4px 16px rgba(0, 0, 0, 0.07), 0px 12px 32px rgba(0, 0, 0, 0.15), 0px 24px 64px rgba(0, 0, 0, 0.2)',
+  button: '0px 2px 0px #C2BF00',
```

Comment:
I'd doble check the as24/ms24 shadows and let you know if we can use a color that we already have in our palette.
Please wait for my inputs before merging this PR. Thanks 😊

Reply:
After checking the shadow color with Sergiu, we can adjust the it with the colors that we already have in the palette:

- as24: auto-300
- ms24: moto-600

The figma is already updated as well.

Reply:
Hey, thanks for checking with UX.

According to [this answer](https://stackoverflow.com/a/3012987), we have no way to set the Shadow color separately. Therefore, I'd recommend to use the compiled CSS variable name from Chakra to solve this.

Let's say we have a color name `brand.btnShadow` in our color pallete, then Chakra will compiled it to a CSS variable name `--chakra-colors-brand-btnShadow`.

I will use this CSS variable name to add flexibility to our custom `boxShadown` for button, like this:

```ts
const shadows = {
  // code omitted
  button: '0px 2px 0px var(--chakra-colors-brand-btnShadow);',
};

export default shadows;

```

A new color name, in `brand` palette, named `btnShadow`. Its value will depends on the selected theme, `brand.300` (For AS24) or `brand.600` (For MS24).

I also thought about the semantic token but it didn't resolve the dynamic color value for `boxShadow`. What do you think?

Would be great if you can share your thoughts to @Averethel @lkappeler

Reply:
Sounds like a solid solution to me 👍